src/main/features/response/views/your-mobile.njk

```diff
@@ -1,14 +1,14 @@
 {% extends "layout.njk" %}
 {% from "form.njk" import csrfProtection, input, saveAndContinueButton, errorSummary %}
 
-{% set heading = 'Add a phone number (optional)' %}
+{% set heading = 'Enter a phone number (optional)' %}
```

Comment:
same here, will need to change in route test as well

Reply:
done